homeassistant/components/tellduslive/sensor.py

```diff
@@ -85,6 +132,11 @@ def _type(self):
         """Return the type of the sensor."""
         return self._id[1]
 
+    @cached_property
```

Comment:
Out of curiosity, why do we want to use `@cached_property` here, the dict access should be O(1)?

Reply:
`cached_property` is slightly faster once the attribute is accessed multiple times. In the end, it probably won't matter much though.
```py
from functools import cached_property
import timeit

z = {"a": 1, "b": 2, "c": 3, "d": 4, "e": 5}

class A:
    @cached_property
    def cache(self):
        return z["d"]

a = A()

def func1():
    for _ in range(10):
        a.cache

def func2():
    for _ in range(10):
        z["d"]

timeit.timeit(lambda: func1, number=1_000_000)  # 0.12474451500000328
timeit.timeit(lambda: func2, number=1_000_000)  # 0.1354764119999743